Dockerfile

```diff
@@ -10,10 +10,8 @@ RUN apk --no-cache add build-base git; \
   nodejs-npm \
```

Comment:
I've not been including `nodejs-npm` in my packages, what am I missing out on? :D

Reply:
News to me but I'll take it! I can just leave it off with no issue? I'll test it of course but I'm a tiny bit suspicious until I can figure out what I'm missing...

Reply:
I think you can remove it :)